### PR TITLE
Pioneer 15 cruise update

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated CruiseInformation.csv with start/end times of Pioneer 15 cruise.

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -282,4 +282,4 @@ PS20200917,R/V Pacific Storm,2020-09-17T17:30:00,2020-09-20T17:30:00,Endurance C
 TS20200925,F/V Tenacious II,2020-09-25T21:00:00,2020-09-26T15:00:00,Pioneer 14c Glider deployments
 ZP20200930,R/V Zephyr,2020-09-30T17:00:00,2020-10-03T23:59:59,Papa 7a Glider deployments
 EN657,R/V Endeavor,2020-10-13T13:00:00,2020-10-18T16:00:00,Pioneer glider cp_340 recovery
-AR48,R/V Neil Armstrong,2020-10-28T00:00:00,2020-11-11T23:59:00,Pioneer 15
+AR48,R/V Neil Armstrong,2020-10-28T13:34:00,2020-11-11T16:36:00,Pioneer 15
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:285]: Looks good.

**Review by snwhite — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[Pioneer 15 AM Updates [DT-102]](https://trello.com/c/PK5rFp9p/102-pioneer-15-am-updates-dt-102)

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (batryn) [reply cruise/CruiseInformation.csv:285]: Looks good

**Review by batryn — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (reedan88) [reply cruise/CruiseInformation.csv:285]: Looks good (Jennifer just beat me to it!)

**Review by reedan88 — APPROVED** [review]

**Pull request merged**